src/librustdoc/config.rs

```diff
@@ -156,6 +156,8 @@ crate struct Options {
     crate run_check: bool,
     /// Whether doctests should emit unused externs
     crate json_unused_externs: bool,
+    /// Capture stdout/stderr of tests.
```

Comment:
This is a little confusing, because `nocapture` is the negation of this.

```suggestion
    /// Whether to skip capturing stdout and stderr of tests.
```

Reply:
Agreed, it looks better!